log/README.md

```diff
@@ -6,7 +6,11 @@ It can be used for both typical application log events, and log-structured data
 
 ## Rationale
 
-TODO
+Logging is an important piece in almost any application, but it's a fundamental piece in distributed applications.
+
+Logs allow to track the system activity allowing to know the operations, errors, unlikely results, etc., which have happened at a specific time, with the possibility to be analyzed in the future for identifying system errors, security issues and so on.
+
+Structure logging allows to register discrete fields in addition to information messages, facilitating their process and storage, aiming to ease their post analysis.
```

Comment:
The first two sections are a bit too generic. We don't need to justify the practice of logging generally, as I hope everyone understands what it is and why it's good! :) Rather we should justify specifically what logging can and should be used for (see [this article](http://peter.bourgon.org/blog/2016/02/07/logging-v-instrumentation.html) and [this article](https://blog.raintank.io/logs-and-metrics-and-graphs-oh-my/) for my thinking here) and why consumers should choose kit/log over other log packages. Your third touches on that: structure_d_ logging is a bit part of it, but I think not the only reason.